Src/Support/Google.Apis.Auth.Tests/Google.Apis.Auth.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <!-- net452 is the earliest desktop version supported by xunit runner -->
+    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="xunit" Version="2.3.0-*" />
+    <PackageReference Include="Xunit.Combinatorial" Version="1.2.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0-*" />
```

Comment:
This package is still in beta, as is xunit 2.3.0, this just gets the latest beta.
These two package references were generated by "dotnet new" generating an xunit test project.